### ISSUE TITLE
Fix Cherry so one does not enter the blanking pit of death

### ISSUE THREAD

**Issue by jpw547** [byuoitav/ui#88]
I selected "Blank" while using ITB-1108B-CP1 as the option on D3. I selected it a second time (while it was already blanked) and for some reason it showed the Via as the current input (though I believe that was the previous input) while the spinners showed around the Blank input button. This did not unblank the screen. I tried to repeat that process to see if it highlighted Via again, but now it just shows the spinners around Blank, and then reselects it. This did not unblank the system, though I don't believe that it should.

I then tried switching inputs. I could choose any input and the UI would reflect my choice appropriately. The object coming from the back end even says that the input is set to the one that I have chosen. However the TV itself refuses to unblank, and is now stuck in the blanking pit of death. The only other option that I can think might be happening is that the TV somehow turned off in all of this madness, but that is not desired behavior either.

**TL;DR** - Make it so that inputs cannot be selected after having already selected them on Cherry.
